src/focus_manager/txm_focus_manager.js

```diff
@@ -262,7 +262,8 @@ export class TXMFocusManager {
         }
 
         let focus = this.currentFocus;
-        if (!focus) {
+        // if no element is currently focused, and the user is attempting to navigate or select, set default focus
+        if (!focus && (inputActions.isMovementAction(action) || inputAction == inputActions.select)) {
```

Comment:
|| inputAction == inputActions.select)....

so users will be accidentally close the ad if they slam on the select button, without thinking?

Reply:
I don't think so, since the action is swallowed, only the focus is set
but if users blindly mash on the select button repeatly, things are bound to go wrong, but really that is true anyway if you think about it

my actual opinion is that this ad as a somewhat poor UX by showing a page that has no focus for too long

remote-control based apps are supposed to follow the principle of always having a clear focus so the user knows what their actions are being applied to